sources/models/src/metal-k8s-1.28/mod.rs

```diff
@@ -13,7 +13,7 @@ use crate::{
 // that uses its name in serialization; internal structures use the field name that points to it
 #[model(rename = "settings", impl_default = true)]
 struct Settings {
-    motd: String,
+    motd: settings_extension_motd::MotdV1,
```

Comment:
Do we need to convert all other top-level fields in these models to settings extensions before we can get rid of the overarching model?